src/components/Button/tests/Button.test.tsx

```diff
@@ -277,40 +254,35 @@ describe('<Button />', () => {
 
   describe('onKeyPress()', () => {
     it('is called when a keypress event is registered on the button', () => {
-      const fakeEventData = {key: 'foo'};
       const spy = jest.fn();
-      shallowWithAppProvider(<Button onKeyPress={spy}>Test</Button>).simulate(
```

Comment:
Simulate was causing this test and the two below to stall and run indefinitely so I choose to use `trigger` instead